cmd/kube-proxy/app/server.go

```diff
@@ -127,16 +130,24 @@ func (s *ProxyServer) Run(_ []string) error {
 		glog.V(2).Infof("Running in resource-only container %q", s.ResourceContainer)
 	}
 
+	if s.Master != "" {
+		if len(s.APIServerList) != 0 {
+			glog.Warningf("Both --mater and --api-servers set. Using --api-servers")
```

Comment:
I'd suggest an error message here?  Why would we ever want anyone to use the word `--master`  when they already know `--api-servers`:) ?